pkg/run/run.go

```diff
@@ -338,10 +350,45 @@ func (r *RunOptions) createConfigMap(kclient kubernetes.Interface, sclient sonob
 	return nil
 }
 
+// processManifestTemplates processes go template variables in the manifest which map to variable in RunOptions
+func (r *RunOptions) processManifestTemplates(manifest *manifest.Manifest) error {
+
+	var templatedPtrs []*string
+
+	templatedPtrs = append(templatedPtrs, &manifest.Spec.Image)
+	for idx, _ := range manifest.PodSpec.Containers {
+		templatedPtrs = append(templatedPtrs, &manifest.PodSpec.Containers[idx].Image)
+	}
+
+	for _, templatedPtr := range templatedPtrs {
+		imageTemplate, err := template.New("manifest").Parse(*templatedPtr)
+		if err != nil {
+			return errors.Wrapf(err, "unable to parse manifest %s", manifest.SonobuoyConfig.PluginName)
+		}
+		var imageBuffer bytes.Buffer
+		err = imageTemplate.Execute(&imageBuffer, r)
+		if err != nil {
+			return errors.Wrapf(err, "unable to update manifest %s", manifest.SonobuoyConfig.PluginName)
+		}
+		*templatedPtr = imageBuffer.String()
+		log.Debugf("manifest %s references image %s", manifest.SonobuoyConfig.PluginName, *templatedPtr)
+	}
+	return nil
+
+}
+
 // Run setup and provision the certification environment.
 func (r *RunOptions) Run(kclient kubernetes.Interface, sclient sonobuoyclient.Interface) error {
 	var manifests []*manifest.Manifest
 
+	imageRepository := "quay.io/ocp-cert"
```

Comment:
We could define a const for this value. See [where we used to create it](https://github.com/redhat-openshift-ecosystem/provider-certification-tool/blob/main/pkg/types.go#L3-L15)